core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -78,6 +78,9 @@
   {% if not iframed %}
     <attribution-guide></attribution-guide>
   {% endif %}
+  {% if nav_mode in ['explore'] %}
+    <exploration-footer></exploration-footer>
+            {% endif %}
```

Comment:
This seems to have worked! Will you double check where I put this is what you were thinking?